README.md

```diff
@@ -1,2 +1,57 @@
 # Volibri
 Version Control for Kolibri UI/UX design
+
+
+**The problems:**
+
+It appears to us that a version control solution is needed to accommodate the collaborative design work we've done for Kolibri.
+There are 3 major issues we want to address with this version control solution:
+
+1. Be able to easily trace back the design decision making history. Following describes a typical issue we are facing more and more often. After many iterations for an UI element, it becomes unclear to us how we arrived at the current approach, and start bringing arguments up that later we realized were fully discussed before.
+
+2. Be able to keep design modifications in sync across the team. Our design team has collectively experienced this pain that if one of us updates the design of a shared UI element, all of us have to find the files that uses this shared element and manually update it.
+
+3. Be able to asynchronously comment on and reply to new designs. We used to post designs in our online chat room to get feedbacks. But just like design takes inspiration, criticizing design also takes inspiration. Since we cannot guarantee to get inspired in time, let's do it asynchronously via Pull Request.
```

Comment:
Alternative way of framing this? Was unclear at first. 
From discussion: cite example of design discussion being pushed up and becoming irrelevant in the Slack #design room